### PR TITLE
update OWNERS files to use teams with active members

### PR DESCRIPTION
**Pull request by dprotaso**
- Eventing & Serving Writers can now approve more packages (in order to share the ownership)
- API packages require approval from Serving/Eventing WG Leads
- Drop individual user names as entries - in favour of using `-writer` and `-reviewer` teams

### CODE REVIEW

**Comment by dprotaso** [comment]
/hold

Would like to get buy in from eventing folks as they'll start getting more notifications for reviews

/assign @psschwei @pierDipi

File: controller/OWNERS [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # The OWNERS file is used by prow to automatically merge approved PRs.
 
 approvers:
-- pkg-controller-writers
+- serving-writers
 
 reviewers:
-- pkg-controller-reviewers
+- serving-reviewers
```
Comment (dprotaso) [inline controller/OWNERS:7]: I'm hoping prow is smart enough to rotate through this list of folks for reviews and approvals vs. the larger pool at the top OWNERS files

**Review by psschwei — COMMENTED** [review]
looks good to me, but will let @pierDipi give the final stamp for eventing side

File: apis/OWNERS [hunk 2]
```diff
@@ -1,9 +1,12 @@
 # The OWNERS file is used by prow to automatically merge approved PRs.
 
 approvers:
-- mattmoor
-- vaikas
-- n3wscott
+- technical-oversight-committee
+- serving-wg-leads
```
Comment (pierDipi) [inline apis/OWNERS:6]: Why isn't eventing-wg-leads here too?

**Review by pierDipi — COMMENTED** [review]
This makes a lot of sense to me, thanks for doing it!

I left just one question

File: apis/OWNERS (hunk 2 above)
Comment (dprotaso) [reply apis/OWNERS:6]: let me add it

**Pull request merged**